src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Avatar
+description: >
+  Avatars display typography, icons, or images in different sizes. They help users quickly recognize
+  an entity.
+package: '@zendeskgarden/react-avatars'
+propSheets:
+  - avatars/src/elements/Avatar.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To visually represent a person, brand, or product</li>
+    </ul>
+  </Use>
+</Usage>
+
+## Examples
+
+### Shape
+
+An Avatar can be a circle or rounded square. View the shape usage [best practices](#best-practices).
+
+import AvatarShape from '../../examples/avatar/AvatarShape.tsx';
+import AvatarShapeCode from '!!raw-loader!../../examples/avatar/AvatarShape.tsx';
+
+<CodeExample code={AvatarShapeCode}>
+  <AvatarShape />
+</CodeExample>
+
+### Size
+
+Sizing can be adjusted to displayed sizes. The default size of an Avatar is `medium`.
+
+import AvatarSize from '../../examples/avatar/AvatarSize.tsx';
+import AvatarSizeCode from '!!raw-loader!../../examples/avatar/AvatarSize.tsx';
+
+<CodeExample code={AvatarSizeCode}>
+  <AvatarSize />
+</CodeExample>
+
+### Type
+
+An Avatar can contain an icon, image, or text.
+
+import AvatarType from '../../examples/avatar/AvatarType.tsx';
+import AvatarTypeCode from '!!raw-loader!../../examples/avatar/AvatarType.tsx';
+
+<CodeExample code={AvatarTypeCode}>
+  <AvatarType />
+</CodeExample>
+
+### Status
+
+A status ring can be applied to the outside of an Avatar to communicate a user is away, available,
+or active.
+
+import AvatarStatus from '../../examples/avatar/AvatarStatus.tsx';
+import AvatarStatusCode from '!!raw-loader!../../examples/avatar/AvatarStatus.tsx';
+
+<CodeExample code={AvatarStatusCode}>
+  <AvatarStatus />
+</CodeExample>
+
+## Best practices
+
+### Selecting the right Avatar shape
+
+Circular Avatars are used to represent a person because the shape is most often associated with a
+profile picture. Rounded squares are used for system applications like objects, brands, or products.
+This helps the user quickly distinguish between a person or a system when scanning.
+
+<BestPractice>
+  <Do imageSource={props.data.avatarShapeCircle.children[0].publicURL}>
+    Use a circular shape to represent a person.
+  </Do>
+  <Do imageSource={props.data.avatarShapeSquare.children[0].publicURL}>
+    Use the rounded square shape to represent objects, brands, or products.
+  </Do>
+</BestPractice>
```

Comment:
After having seen the design options for this, I think the `<BestPractice>` component is going to need rework. But I don't want to continue to get in the way of this PR. Let's figure it out and plan for later.